raiden/blockchain_events_handler.py

```diff
@@ -226,7 +235,7 @@ def handle_channel_new(raiden: "RaidenService", event: Event):  # pragma: no uni
 
 def create_new_balance_state_change(
     chain_state: "ChainState", event: Event
-) -> Tuple[Optional[ContractReceiveChannelNewBalance], bool]:
+) -> Tuple[Optional[Tuple[ContractReceiveChannelNewBalance, ActionChannelUpdateFee]], bool]:
```

Comment:
This becomes much easier with:
`Tuple[List[StateChange], bool]`.

In the case for:
```
if previous_channel_state is None:
        return None, False
```
becomes
```
if previous_channel_state is None:
        return list(), False
```

So need to use a tuple of optional tuple of two state changes

Reply:
I see your point, but the idea was to make clear that when something is returned, than it's exactly these two state changes.